apps/system/fxa/elements/fxa-email.html

```diff
@@ -3,12 +3,12 @@
     <section class="vertical">
       <div id="fxa-logo" class="fxa-logo"></div>
       <p class="fxa-intro" data-l10n-id="fxa-use-email-to-sign-in">
-        Use your email address to create an account or sign in.
       </p>
       <input id="fxa-email-input" data-l10n-id="fxa-enter-email-input" type="email" placeholder="Enter your email" required="" x-inputmode="verbatim">
 
-      <p id="fxa-notice" class="fine-print">
-        By proceeding, I agree to the <a class="disabled">Terms of Service</a> and <a class="disabled">Privacy Notice</a> of Firefox cloud services.
+      <p id="fxa-notice" class="fine-print" data-l10n-id="fxa-notice2">
```

Comment:
You didn't change the string so I think you don't need to change the string id: `fxa-notice`